pydarn/plotting/maps.py

```diff
@@ -134,6 +142,10 @@ def plot_mapdata(cls, dmap_data: List[dict], ax=None,
             imf_dial: bool
                 If True, draw an IMF dial of the magnetic field clock angle.
                 Default: True
+            reference_vector: bool
+                If True include reference vector and label in
+                lower right corner
+                Default: False
```

Comment:
```suggestion
                If True include reference velocity vector and 
                label in lower right corner
                Default: False
```

or if my above suggestion is used:

```suggestion
                Velocity magnitude to use for the
                reference vector, drawn in lower
                right corner
                Default: 500
```